src/main/java/mate/academy/bookshop/model/Book.java

```diff
@@ -32,4 +36,6 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    @Column(name = "is_deleted", nullable = false, columnDefinition = "boolean default false")

```

Comment:
try to use type `BIT` in the liquibase script for that field and I think it should work without the `columnDefinition`